### PR TITLE
[Forward Port] [#14089] Add Malaysian Locale Code

### PR DESCRIPTION
**Pull request by osrecio**
Add locale code for Malaysian
### Description
Add locale config fix typo for Malaysian

### Fixed Issues (if relevant)

1. magento/magento2#14089: Malaysian (Malaysia) missing from locale list 

### Manual testing scenarios
1. Go to Store -> Configuration-> General -> Locale Options
2. Change locale to Malaysian (Malaysia)

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: lib/internal/Magento/Framework/Locale/Config.php [hunk 1]
```diff
@@ -74,7 +74,7 @@ class Config implements \Magento\Framework\Locale\ConfigInterface
         'lv_LV', /*Latvian (Latvia)*/
         'mk_MK', /*Macedonian (Macedonia)*/
         'mn_Cyrl_MN', /*Mongolian (Mongolia)*/
-        'ms_Latn_MY', /*Malaysian (Malaysia)*/
+        'ms_MY', /*Malaysian (Malaysia)*/
```
Comment (orlangur) [inline lib/internal/Magento/Framework/Locale/Config.php]: I [doubt](https://www.google.com/search?ei=6TK2Wr_5E4b-sQHlj4rwBw&q=locale+ms_Latn_MY&oq=locale+ms_Latn_MY&gs_l=psy-ab.3...10031.10575.0.10662.7.6.0.0.0.0.186.446.2j2.4.0....0...1c.1.64.psy-ab..6.0.0....0.ZHbjyLSAfJM) this is a typo, should we add both locale codes maybe?

**Review by orlangur — CHANGES_REQUESTED** [review]

File: lib/internal/Magento/Framework/Locale/Config.php (hunk 1 above)
Comment (osrecio) [reply lib/internal/Magento/Framework/Locale/Config.php]: I will add in a minute ;-)

Comment (osrecio) [reply lib/internal/Magento/Framework/Locale/Config.php]: Done!

**Review by orlangur — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @orlangur, thank you for the review.
[ENGCOM-1055](https://jira.corp.magento.com/browse/ENGCOM-1055) has been created to process this Pull Request

**Pull request merged**